main.go

```diff
@@ -110,20 +104,33 @@ func buildRsync(rs *Rsync) *exec.Cmd {
 		args = append(args, fmt.Sprintf("--exclude=%s", rs.Exclude))
 	}
 
-	args = append(args, rs.Source)
-	args = append(args, fmt.Sprintf("%s@%s:%s", rs.User, rs.host, rs.Target))
+	srcs, err := filepath.Glob(path.Join(root, rs.Source))
```

Comment:
this should account for the possibility that `rs.Source` could be an absolute path. In some cases I generate files in `/drone/tmp` and then upload from there